app/views/objects/_sidebar.html.slim

```diff
@@ -15,17 +15,17 @@ div class="hidden lg:block w-1/4"
                 = render "objects/sidebar/section/link",
                     title: included_module.constant,
                     href: object_path(object: included_module.path)
-      - unless @object.ruby_methods.select(&:class_method?).empty?
+      - unless @object.ruby_class_methods.empty?
         = render "objects/sidebar/section", title: 'Class Methods' do
           ul class="font-mono text-sm"
-            - @object.ruby_methods.select(&:class_method?).sort_by(&:name).each do |m|
+            - @object.ruby_class_methods.each do |m|
```

Comment:
Can we skip rendering these blocks if there are no class/instance methods, it looks a bit odd showing the header with no methods following?

<img width="1394" alt="Screen Shot 2020-09-01 at 11 05 21 pm" src="https://user-images.githubusercontent.com/996377/91854813-a202af80-eca7-11ea-96b4-22063fc89e8f.png">

Reply:
This is incorrect place of code to comment this issue:

![image](https://user-images.githubusercontent.com/6510020/92516022-3b3c4380-f21d-11ea-9ee6-d26a13c3a3e6.png)

(I was confused, I see a check here already, but I could reproduce this issue locally),

but then I've realized that this is `sidebar` and have added such conditions to the `show` page.

Thank you.